cmake/modules/Findfuse3.cmake

```diff
@@ -0,0 +1,34 @@
+# This module can find FUSE Library
```

Comment:
instead of replicating `Findfuse.cmake`, i think a better approach is to add fuse3 as another suffix when finding FUSE library. could you help test and review #34604 ?

Reply:
sure, will test it.